### PR TITLE
Slvnperron patch 1

### PR DESCRIPTION
**Pull request by slvnperron**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,11 +142,11 @@
 				</div>
 				<div class="col-md-6">
 					<div class="row">
-						<p class="pull-right">Jump in the train</p>
+						<p class="pull-right">Hop in the train</p>
```
Comment (ghost) [inline index.html:145]: Hop on

Comment (EFF) [inline index.html:145]: +1

File: index.html [hunk 2]
```diff
@@ -25,11 +25,11 @@
 						<div class="register-panel pull-right">
 							<h3>Get full access before everyone</h3>
 							<ul>
-								<li>Get access to the private beta</li>
-								<li>Get 100$ of free cloud credits on public release</li>
-								<li>Access to the internal development roadmap</li>
-								<li>Get a larger free tier on public release</li>
-								<li>Free access to all of invoked.io events, conferences and meetups</li>
+								<li>Get access to the <b>private beta</b></li>
+								<li>Get 100$ of free <b>cloud credits</b> on public release</li>
```
Comment (EFF) [inline index.html:29]: - get a 100$ credits on public release

and i would remove the "get a larger free tier on public release"

**Comment by osylvain** [comment]
Since github can't merge automatically and you had only a few changes, I added them to my PR so you wont have to merge manually. That's why i'm closing this PR.

**Comment by slvnperron** [comment]
thanks man

**Pull request closed without merging**